src/main/java/uk/gov/companieshouse/model/Attachment.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.companieshouse.model;
+
+import java.util.Objects;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+public class Attachment {
+
+    @NotBlank(message = "attachement id must not be null.")
```

Comment:
Can you rename this to 'id must not be blank' to be consistent with the other messages?

Reply:
I am not convinced to having periods at the end, never seen error messages in response bodies with dots.